### DISCUSSION TITLE
Load fragment on page load

### DISCUSSION THREAD

**Discussion by alexandrefjacques** [unpoly/unpoly#312] [General]
Hi,

Is it possible to, on page load, make a fragment updates itself? I would be like a [up-poll] triggered once after page load.

TIA.

**Comment by adam12** [comment]
Perhaps a custom compiler?

```javascript
up.compiler('[reload]', (el) => {
  up.reload(el)
})
```

```html
<div id="reload-me" reload>
  I'll be reloaded on page load
</div>
```

**Reply by alexandrefjacques** [reply]
Great. Thanks!

**Reply by pySilver** [reply]
How do I define a source url that way?

**Reply by adam12** [reply]
@pySilver Add an [`up-source`](https://unpoly.com/up-source) attribute to the element being reloaded.

**Comment by pfw** [comment]
I would think the most straightforward way to do that would be calling up.follow in JS on the when document ready event has fired.  

Peter W.

**Comment by pfw** [comment]
Or Adam’s method. All the bits are available in the API just not out of the box as attributes. 

Peter W.
